specification/mediaservices/resource-manager/readme.md

```diff
@@ -62,11 +62,27 @@ directive:
     from: Accounts.json
 ```
 
+
+### Tag: package-2022-08
+
+These settings apply only when `--tag=package-2022-08` is specified on the command line.
+
+```yaml $(tag) == 'package-2022-08'
+input-file:
+  - Microsoft.Media/stable/2022-08-01/AccountFilters.json
+  - Microsoft.Media/stable/2021-11-01/Accounts.json
+  - Microsoft.Media/stable/2022-08-01/AssetsAndAssetFilters.json
+  - Microsoft.Media/stable/2022-08-01/ContentKeyPolicies.json
+  - Microsoft.Media/stable/2021-11-01/Encoding.json
+  - Microsoft.Media/stable/2022-08-01/StreamingPoliciesAndStreamingLocators.json
+  - Microsoft.Media/stable/2021-11-01/streamingservice.json
```

Comment:
@seaeunlee14  or @ZzzMao  can you guys do a rebase with main branch, Stephanie has checked in her changes, and we can start using the 2022-08-01 version of streaming swagger in your readme changes.